core/src/main/java/com/linecorp/armeria/common/util/SystemInfo.java

```diff
@@ -154,8 +167,28 @@ public static long currentTimeMicros() {
         return JavaVersionSpecific.get().currentTimeMicros();
     }
 
-    private static boolean isLinux() {
-        return Ascii.toLowerCase(System.getProperty("os.name", "")).startsWith("linux");
+    /**
+     * Returns the operating system type.
+     */
+    public static OsType osType() {
+        return osType;
+    }
+
+    /**
+     * Operating system type enumeration value
```

Comment:
```suggestion
     * Operating system.
```

(type enumeration value is obvious from it being `enum` so no need to repeat in javadoc)